src/main/java/com/google/devtools/build/lib/packages/BuiltinRestriction.java

```diff
@@ -95,12 +93,21 @@ public static void failIfCalledOutsideAllowlist(
    */
   public static void failIfModuleOutsideAllowlist(
       BazelModuleContext moduleContext, Collection<AllowlistEntry> allowlist) throws EvalException {
-    if (moduleContext.label().getRepository().getName().equals("_builtins")) {
+    failIfLabelOutsideAllowlist(moduleContext.label(), moduleContext.repoMapping(), allowlist);
+  }
+
+  /**
+   * Throws {@code EvalException} if the given {@link Label} is not within either 1) the builtins
+   * repository, or 2) a package or subpackage of an entry in the given allowlist.
+   */
+  public static void failIfLabelOutsideAllowlist(
+      Label label, RepositoryMapping repoMapping, Collection<AllowlistEntry> allowlist)
+      throws EvalException {
+    if (label.getRepository().getNameWithAt().equals("@_builtins")) {
```

Comment:
This seems to have lost the changes from https://github.com/bazelbuild/bazel/commit/c4560822cc9d48f6642fd946ae0eac93a40668cc#diff-b3a4058cfe39033fa0d1ce54e85a94b280c1a34b9e8bb05d3c809ea3496388a2

@comius

Reply:
Thanks for the hint!